groovy-android-gradle-plugin/src/test/groovy/groovyx/functional/FullCompilationSpec.groovy

```diff
@@ -27,7 +27,7 @@ import static groovyx.internal.TestProperties.compileSdkVersion
  * Complete test suite to ensure the plugin works with the different versions of android gradle plugin.
  * This will only be run if the system property of 'allTests' is set to true
  */
-@IgnoreIf({ !allTests })
+@IgnoreIf({ false })
```

Comment:
No good reason, it was accident. I'll revert that. I wanted to make sure that this tests run locally, but unfortunately I did up committing that.